.github/workflows/botocore-sync.yml

```diff
@@ -12,8 +12,8 @@ jobs:
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v4
         with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          role-to-assume: ${{ secrets.CODEBUILD_ROLE_ARN }}
+          role-duration-seconds: 10800
```

Comment:
How did we get this number

Reply:
It is the max session duration that was set through CDK